pkg/kubelet/kubelet.go

```diff
@@ -2735,14 +2738,23 @@ func (kl *Kubelet) setNodeStatusMachineInfo(node *api.Node) {
 	// TODO: Post NotReady if we cannot get MachineInfo from cAdvisor. This needs to start
 	// cAdvisor locally, e.g. for test-cmd.sh, and in integration test.
 	info, err := kl.GetCachedMachineInfo()
+	gpuCount := 0
+
+	for _, plugin := range kl.gpuPlugins {
+		currCount, err := plugin.Capacity()
```

Comment:
Actually, this code really need to be changed, and I want to support multiple GPU vendors. Each vendor should have its own plugin. I would like to just put Nvidia GPU check here.